CONTRIBUTING.md

```diff
@@ -104,16 +105,35 @@ Useful links:
 The simplest way to check how documentation is rendered is to make a pull request. 
 CI will build it, publish and attach a link to the pull request.
 
-#### 7.1 Adding tutorials (optional)
+#### 7.1 Standard scenarios
```

Comment:
We should somehow highlight the scenarios names, now it is hard to find the right one